apps/server/src/modules/tldraw/service/tldraw.ws.service.ts

```diff
@@ -0,0 +1,208 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { TldrawConfig } from '@src/modules/tldraw/config';
+import { Persitence, WSConnectionState, WSMessageType, WsSharedDocDo } from '@src/modules/tldraw/types';
+import WebSocket from 'ws';
+import { applyAwarenessUpdate, encodeAwarenessUpdate, removeAwarenessStates } from 'y-protocols/awareness';
+import { encoding, decoding, map } from 'lib0';
+import { readSyncMessage, writeSyncStep1, writeUpdate } from 'y-protocols/sync';
+import { TldrawBoardRepo } from '@src/modules/tldraw/repo';
+
+@Injectable()
+export class TldrawWsService {
+	pingTimeout: number;
+
+	persistence: Persitence | null = null;
+
+	docs = new Map();
+
+	constructor(
+		readonly configService: ConfigService<TldrawConfig, true>,
+		private readonly tldrawBoardRepo: TldrawBoardRepo
+	) {
+		this.pingTimeout = this.configService.get<number>('TLDRAW_PING_TIMEOUT') ?? 10000;
```

Comment:
Can you move the default value to config/default.schema.json ? It exists no need to add it on this place.